src/app/app-main/app-main.tsx

```diff
@@ -9,6 +9,9 @@ import { Button } from 'react-bootstrap';
 import AppAuthenticated from '../app-authenticated/app-authenticated';
 import AppPublic from '../app-public/app-public';
 import './app-main.module.css';
+import WorkPackageDetails from '../../components/work-package-details/work-package-details';
+import { exampleWorkPackage1 } from 'utils';
+import WorkPackageSummary from '../../components/work-package-summary/work-package-summary';
```

Comment:
As a general rule of thumb for consistency / simplicity and keeping it clean, I try to organize the imports at the top of every file in importance and external-ness order: external packages, utils package, relative path imports, and CSS always last. I find this makes it easier to read, but it's up to you.